README.md

```diff
@@ -1,6 +1,7 @@
 # Bynder C# SDK
 
-[![Build Status](https://travis-ci.org/Bynder/bynder-c-sharp-sdk.svg?branch=master)](https://travis-ci.org/Bynder/bynder-c-sharp-sdk)
+![Tests](https://github.com/Bynder/bynder-c-sharp-sdk/workflows/Tests/badge.svg)
+![Publish](https://github.com/Bynder/bynder-c-sharp-sdk/workflows/Publish/badge.svg)
```

Comment:
What should these urls point to? They don't seem to work for me.

Reply:
Nevermind I get it after reading this https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge

Reply:
Yeah that was to prep the badge to the Bynder master repo.